site2/docs/helm-deploy.md

```diff
@@ -1,17 +1,20 @@
 ---
 id: helm-deploy
-title: Deploy Pulsar cluster using Helm
-sidebar_label: "Deployment"
+title: Deploy a Pulsar cluster on Kubernetes
+sidebar_label: "Deploy"
 ---
 
-Before running `helm install`, you need to decide how to run Pulsar.
-Options can be specified using Helm's `--set option.name=value` command line option.
+Before deploying a Pulsar cluster, you need to [prepare Kubernetes resources](helm-prepare.md) and then continue with the following steps.
+1. Select configuration options
+2. Install dependent charts
+3. Prepare Helm release
+4. Deploy your Pulsar cluster using Helm
 
 ## Select configuration options
 
-In each section, collect the options that are combined to use with the `helm install` command.
+Specify how to run Pulsar using Helm's `--set option.name=value` command line option. In each section, collect the options that are combined to use with the `helm install` command.
 
-### Kubernetes namespace
+#### Kubernetes namespace
```

Comment:
Why use #### instead of ###? Is this an intended behavior for not showing the heading on the right navigation?

Reply:
Yes, that's my intention to make the hot links simple and clean for general steps.